authentication-typescript/template/src/pages/home.tsx

```diff
@@ -64,7 +64,7 @@ export const HomePage = (): JSX.Element => {
     const theme = useTheme();
     const classes = useStyles(theme);
     const { setDrawerOpen } = useDrawer();
-    const xs = useMediaQuery(theme.breakpoints.down('xs'));
+    const xs = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
rename this variable to reflect what it's actually doing. If it's no longer responding at xs let's not call it xs.